### ISSUE TITLE
Badge/Shield

### ISSUE THREAD

**Issue by utdemir** [keybase/keybase-issues#849]
Hi. I think it'd be very useful to have a keybase badge, something like:

![](http://img.shields.io/badge/Public%20Key-available-green.svg)

with a link to Keybase profile, or a more detailed one including proofs etc. Or it can be easily implemented by a third party too, but a standart solution would be better.

**Comment by bxm156** [comment]
+1. It would also be nice if we could have a web badge that we could post on our personal website, and have it link to our keybase profile

**Comment by mendhak** [comment]
Other ideas:

![](https://img.shields.io/badge/Keybase-mendhak-blue.svg)

or

![](https://img.shields.io/badge/PGP-76CBE9A9-blue.svg)


+1 on this anyway - I sign my releases and would like downloaders to verify or be aware I'm on Keybase.

**Comment by amosshapira** [comment]
Yep - I came here to suggest just that. My suggested use case is to let people add this badge to their e-mail signature, same way that I do with my LinkedIn profile. Here is LinkedIn's page about it, for more ideas: https://www.linkedin.com/profile/profile-badges

**Comment by yzgyyang** [comment]
Hi - I would like to have that as well.

**Comment by yzgyyang** [comment]
Hey everyone - I am trying to add Keybase support in https://github.com/badges/shields/issues/1299. Could anyone interested add comments to the issue linked about what kinds of badges are wanted for Keybase, and I will try adding it. Thanks!

**Comment by maxtaco** [comment]
cc @malgorithms

**Comment by lukechilds** [comment]
I would recommend using the 64-bit PGP fingerprint, 32-bit PGP fingerprint's are trivial to brute force: https://evil32.com

Something like:

<a title="Keybase" target="_blank" href="https://keybase.io/hyperdex">
  <img src="https://img.shields.io/badge/PGP-31DB%20EB37%20BF19%209A3C-green.svg">
</a>

**Comment by lukechilds** [comment]
I've implemented this in https://badgen.net: https://github.com/amio/badgen-service/pull/141

**Comment by SkypLabs** [comment]
I've done the same for [Shields](https://shields.io/): badges/shields#2969
